### ISSUE TITLE
[Bug?]: Studio fails to start because of wrong Prisma runtime

### ISSUE THREAD

**Issue by dennemark** [redwoodjs/redwood#10433]
### What's not working?

Hi,
I have an issue when trying to start redwood studio. I added the binaryTarget to my schema.prisma and ran `yarn rw prisma generate`, but `yarn rw studio` still fails with the attached error message. Does studio use a different schema file?
I am on Ubuntu 23.1. `yarn rw dev` etc. works


```bash
$ yarn rw studio
{"level":30,"time":1712653597066,"pid":100304,"hostname":"MYHOST","msg":"Starting RedwoodJS Studio"}
{"level":30,"time":1712653597073,"pid":100304,"hostname":"MYHOST","msg":"Migrating local Prisma database"}
/home/MYPATH/node_modules/@redwoodjs/studio/api/db/client/runtime/library.js:67
${at(e)}`}var $d=$("prisma:client:engines:resolveEnginePath"),qd=()=>new RegExp("runtime[\\\\/]library\\.m?js$");async function dl(e,t){let r={binary:process.env.PRISMA_QUERY_ENGINE_BINARY,library:process.env.PRISMA_QUERY_ENGINE_LIBRARY}[e]??t.prismaPath;if(r!==void 0)return r;let{enginePath:n,searchedLocations:i}=await jd(e,t);if($d("enginePath",n),n!==void 0&&e==="binary"&&Di(n),n!==void 0)return t.prismaPath=n;let o=await ft(),s=t.generator?.binaryTargets??[],a=s.some(d=>d.native),l=!s.some(d=>d.value===o),u=__filename.match(qd())===null,c={searchedLocations:i,generatorBinaryTargets:s,generator:t.generator,runtimeBinaryTarget:o,queryEngineName:ml(e,o),expectedLocation:vr.default.relative(process.cwd(),t.dirname),errorStack:new Error().stack},p;throw a&&l?p=ul(c):l?p=al(c):u?p=ll(c):p=cl(c),new k(p,t.clientVersion)}async function jd(engineType,config){let binaryTarget=await ft(),searchedLocations=[],dirname=eval("__dirname"),searchLocations=[config.dirname,vr.default.resolve(dirname,".."),config.generator?.output?.value??dirname,vr.default.resolve(dirname,"../../../.prisma/client"),"/tmp/prisma-engines",config.cwd];__filename.includes("resolveEnginePath")&&searchLocations.push(Qs());for(let e of searchLocations){let t=ml(engineType,binaryTarget),r=vr.default.join(e,t);if(searchedLocations.push(e),pl.default.existsSync(r))return{enginePath:r,searchedLocations}}return{enginePath:void 0,searchedLocations}}function ml(e,t){return e==="library"?Xr(t,"fs"):`query-engine-${t}${t==="windows"?".exe":""}`}function Vn(e,t){return{batch:e,transaction:t?.kind==="batch"?{isolationLevel:t.options.isolationLevel}:void 0}}var fo=F(Bi());function fl(e){return e?e.replace(/".*"/g,'"X"').replace(/[\s:\[]([+-]?([0-9]*[.])?[0-9]+)/g,t=>`${t[0]}5`):""}function gl(e){return e.split(`
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    ^

PrismaClientInitializationError: Prisma Client could not locate the Query Engine for runtime "debian-openssl-3.0.x".

This happened because Prisma Client was generated for "darwin-arm64", but the actual deployment required "debian-openssl-3.0.x".
Add "debian-openssl-3.0.x" to `binaryTargets` in the "schema.prisma" file and run `prisma generate` after saving it:

generator client {
  provider      = "prisma-client-js"
  binaryTargets = ["native", "debian-openssl-3.0.x"]
}

The following locations have been searched:
  /home/MYPATH/node_modules/@redwoodjs/studio/api/db/client
  /Users/tobbe/dev/redwood/studio/api/db/client
  /home/MYPATH/node_modules/@redwoodjs/studio/api/.prisma/client
  /tmp/prisma-engines
  /home/MYPATH/node_modules/@redwoodjs/studio/api/db
    at dl (/home/MYPATH/node_modules/@redwoodjs/studio/api/db/client/runtime/library.js:67:805)
    at async Object.loadLibrary (/home/MYPATH/node_modules/@redwoodjs/studio/api/db/client/runtime/library.js:114:9816)
    at async zt.loadEngine (/home/MYPATH/node_modules/@redwoodjs/studio/api/db/client/runtime/library.js:116:448)
    at async zt.instantiateLibrary (/home/MYPATH/node_modules/@redwoodjs/studio/api/db/client/runtime/library.js:115:10322) {
  clientVersion: '5.7.0',
  errorCode: undefined
}

Node.js v20.11.0
```

I guess the mentioning of: `/Users/tobbe/dev/redwood/studio/api/db/client` should also not happen @Tobbe ?

### How do we reproduce the bug?

_No response_

### What's your environment? (If it applies)

```shell
System:
    OS: Linux 6.5 Ubuntu 23.10 23.10 (Mantic Minotaur)
    Shell: 5.2.15 - /bin/bash
  Binaries:
    Node: 20.11.0 - /tmp/xfs-72ecbc15/node
    Yarn: 3.2.3 - /tmp/xfs-72ecbc15/yarn
  npmPackages:
    @redwoodjs/auth-custom-setup: 7.3.0 => 7.3.0 
    @redwoodjs/auth-dbauth-setup: 7.3.0 => 7.3.0 
    @redwoodjs/cli-data-migrate: 7.3.0 => 7.3.0 
    @redwoodjs/core: 7.3.0 => 7.3.0 
    @redwoodjs/realtime: 7.3.0 => 7.3.0 
    @redwoodjs/studio: 11 => 11.3.1
```


### Are you interested in working on this?

- [ ] I'm interested in working on this

**Comment by dthyresson** [comment]
Hi @dennemark I think https://github.com/redwoodjs/studio/pull/44 in the Studio repo might fix this -- we just haven't made a release recently as the team's focus has been RSC. @Tobbe and @Josh-Walker-GM am I correct?

**Comment by dennemark** [comment]
🤦‍♂️
should pick the right repository....
closing this.
Thanks!!!

**Comment by Josh-Walker-GM** [comment]
Hey all 👋

We just released `11.4.0` if you could try that out it should fix this problem 🤞

**Comment by dennemark** [comment]
Seems to work! Thank you! Great how studio evolved!
